### DISCUSSION TITLE
Migrate from Home Assistant Addon to Docker on separate machine?

### DISCUSSION THREAD

**Discussion by metheos** [tsightler/ring-mqtt#844] [Support Q&A] [answered]
Curious how it would look or if it's even possible to remove the home assistant addon and instead run the project in Docker on a separate machine without losing all the entities and automations that have already been created. My pi can barely handle the video streams, so I'd like to do this if it wouldn't create a lot of headaches recreating the work I've done already.

**Comment by tsightler** [comment, marked as answer]
You can run ring-mqtt anywhere you want, the device IDs are deterministically generated based on the IDs of the devices themselves so running the code somewhere else doesn't matter, as long as you use the same account to logic.  Of course, for cameras, you'd need to manually update the RTSP URLs to point to the new place, but otherwise, no difference, I regularly shut down my addon and run my local dev branch against my HA instance for testing purposes.

**Reply by metheos** [reply]
Awesome, thanks for the information!
